docs/sources/configuration/_index.md

```diff
@@ -2308,6 +2308,9 @@ The `limits_config` block configures global and per-tenant limits in Loki.
 # CLI flag: -ruler.max-rule-groups-per-tenant
 [ruler_max_rule_groups_per_tenant: <int> | default = 0]
 
+# Ruler alertmanager configuration per tenant.
+[ruler_alertmanager_config: <alertmanager_config>]
```

Comment:
I think this is stale now?

Reply:
What do you mean?

Reply:
for the global config yes, but not for the limits config. This part of the doc is for the limits config

Reply:
Ah! Gotcha